src/repos.json

```diff
@@ -23,42 +23,34 @@
       "silverstripe/vendor-plugin",
       "silverstripe/silverstripe-versioned",
       "silverstripe/silverstripe-versioned-admin",
-      "silverstripe-themes/silverstripe-simple"
+      "silverstripe/silverstripe-simple"
     ]
   },
   {
     "id": "supported",
     "name": "Supported",
     "repos": [
-      "silverstripe/gha-action-ci",
-      "silverstripe/gha-auto-tag",
-      "silverstripe/gha-ci",
-      "silverstripe/gha-dispatch-ci",
-      "silverstripe/gha-gauge-release",
-      "silverstripe/gha-generate-matrix",
-      "silverstripe/gha-issue",
-      "silverstripe/gha-keepalive",
-      "silverstripe/gha-merge-up",
-      "silverstripe/gha-pull-request",
-      "silverstripe/gha-run-tests",
-      "silverstripe/gha-tag-release",
-      "silverstripe/gha-trigger-ci",
-      "silverstripe/gha-update-js",
-      "silverstripe/cow",
-      "silverstripe/rhino",
-      "silverstripe/github-issue-search-client",
-      "silverstripe/module-standardiser",
-      "silverstripe/silverstripe-tx-translator",
-      "silverstripe/markdown-php-codesniffer",
-      "silverstripe/.github",
-      "silverstripe/eslint-config",
-      "silverstripe/webpack-config",
-      "silverstripe/silverstripe-module",
-      "silverstripe/api.silverstripe.org",
-      "silverstripe/doc.silverstripe.org",
-      "silverstripe/silverstripe-userhelp-content",
-      "silverstripe/demo.silverstripe.org",
-      "silverstripe/silverstripe-frameworktest",
+      "silverstripe/developer-docs",
```

Comment:
Shouldn't this list by dynamically read from the supported-modules repo?

Reply:
Not on every request, no. This file gets rebuilt based on the `supported-modules` repo data on each deployment, as per the information in the readme. This is the way it has always worked.
We just need to make sure to trigger a deployment if we update the list.